Formula/dynomite.rb

```diff
@@ -0,0 +1,38 @@
+class Dynomite < Formula
+  desc "Generic dynamo implementation for different k-v storage engines"
+  homepage "https://github.com/Netflix/dynomite"
+  url "https://github.com/Netflix/dynomite/archive/v0.6.22.tar.gz"
+  sha256 "9c3c60d95b39939f3ce596776febe8aa00ae8614ba85aa767e74d41e302e704a"
+  license "Apache-2.0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "openssl@1.1"
+
+  def install
+    system "autoreconf", "-fvi"
+    system "./configure", *std_configure_args,
+                          "--disable-silent-rules",
+                          "--sbindir=#{bin}"
```

Comment:
Why not `sbin`?

Reply:
Updated, but saw a few other formula used this pattern:

```text
grep -r '"--sbindir=#{bin}"' Formula
Formula/fping.rb:    system "./configure", *std_configure_args, "--sbindir=#{bin}"
Formula/cracklib.rb:                          "--sbindir=#{bin}",
Formula/smartmontools.rb:                          "--sbindir=#{bin}",
Formula/bacula-fd.rb:                          "--sbindir=#{bin}",
Formula/burp.rb:                          "--sbindir=#{bin}",
Formula/gnupg@2.2.rb:                          "--sbindir=#{bin}",
Formula/gnupg.rb:                          "--sbindir=#{bin}",
Formula/vnstat.rb:                          "--sbindir=#{bin}",
Formula/httpd.rb:                          "--sbindir=#{bin}",
Formula/ganglia.rb:                          "--sbindir=#{bin}",
```

Reply:
Yea, those are all really old formulae, so I suspect those flags are just from when Homebrew didn't handle `sbin` well.